README.md

```diff
@@ -1,2 +1,127 @@
 # devices
 Servo-specific APIs to access various devices
+
+----------
+
+###How to use
+
```

Comment:
And for the devs maybe an Under the hood or someting, where we list the underlying crates (e.g. linux->blurz).